### PR TITLE
E1743: ignore empty click command (#262)

### PR DESCRIPTION
**Pull request by pcurt**

### CODE REVIEW

File: adapters/ikea/tradfri_switch_on_off.py [hunk 1]
```diff
@@ -18,6 +18,16 @@ def __init__(self, devices):
 
         self.devices.append(self.switch)
 
+    def handleMqttMessage(self, device_data, message):
+        if 'click' not in message.raw:
```
Comment (stas-demydiuk) [inline adapters/ikea/tradfri_switch_on_off.py]: This probably will filter all heartbeat messages if there are any

Comment (pcurt) [reply adapters/ikea/tradfri_switch_on_off.py]: Is there heartbeat messages on E1743 ?  I thought that all messages contain "click"...

Comment (stas-demydiuk) [reply adapters/ikea/tradfri_switch_on_off.py]: Have no idea as doesn't have this device, but usually zigbee devices which work on battery have such messages

**Comment by SargonofAssyria** [comment]
FYI. These are the messages coming from the E1743, when click, hold and release.
`info  2020-03-04 09:10:54: MQTT publish: topic 'zigbee2mqtt/SchakelaarEddy/click', payload 'brightness_down'
info  2020-03-04 09:10:54: MQTT publish: topic 'zigbee2mqtt/SchakelaarEddy', payload '{"linkquality":31,"last_seen":"2020-03-04T09:10:54+01:00","battery":87,"update_available":true,"click":"brightness_down","device":{"friendlyName":"SchakelaarEddy","model":"E1743","ieeeAddr":"0xec1bbdfffe910c8a","networkAddress":34544,"type":"EndDevice","manufacturerID":4476,"manufacturerName":"IKEA of Sweden","powerSource":"Battery","applicationVersion":34,"stackVersion":98,"zclVersion":3,"hardwareVersion":1,"dateCode":"20190410","softwareBuildID":"2.2.008"}}'
info  2020-03-04 09:10:54: MQTT publish: topic 'zigbee2mqtt/SchakelaarEddy', payload '{"linkquality":31,"last_seen":"2020-03-04T09:10:54+01:00","battery":87,"update_available":true,"click":"","device":{"friendlyName":"SchakelaarEddy","model":"E1743","ieeeAddr":"0xec1bbdfffe910c8a","networkAddress":34544,"type":"EndDevice","manufacturerID":4476,"manufacturerName":"IKEA of Sweden","powerSource":"Battery","applicationVersion":34,"stackVersion":98,"zclVersion":3,"hardwareVersion":1,"dateCode":"20190410","softwareBuildID":"2.2.008"}}'
info  2020-03-04 09:10:54: MQTT publish: topic 'zigbee2mqtt/SchakelaarEddy/click', payload 'brightness_stop'
info  2020-03-04 09:10:54: MQTT publish: topic 'zigbee2mqtt/SchakelaarEddy', payload '{"linkquality":31,"last_seen":"2020-03-04T09:10:54+01:00","battery":87,"update_available":true,"click":"brightness_stop","device":{"friendlyName":"SchakelaarEddy","model":"E1743","ieeeAddr":"0xec1bbdfffe910c8a","networkAddress":34544,"type":"EndDevice","manufacturerID":4476,"manufacturerName":"IKEA of Sweden","powerSource":"Battery","applicationVersion":34,"stackVersion":98,"zclVersion":3,"hardwareVersion":1,"dateCode":"20190410","softwareBuildID":"2.2.008"}}'
info  2020-03-04 09:10:54: MQTT publish: topic 'zigbee2mqtt/SchakelaarEddy', payload '{"linkquality":31,"last_seen":"2020-03-04T09:10:54+01:00","battery":87,"update_available":true,"click":"","device":{"friendlyName":"SchakelaarEddy","model":"E1743","ieeeAddr":"0xec1bbdfffe910c8a","networkAddress":34544,"type":"EndDevice","manufacturerID":4476,"manufacturerName":"IKEA of Sweden","powerSource":"Battery","applicationVersion":34,"stackVersion":98,"zclVersion":3,"hardwareVersion":1,"dateCode":"20190410","softwareBuildID":"2.2.008"}}'

`

**Comment by stas-demydiuk** [comment]
Thanks for the contribution!

**Pull request merged**